extension/content_script.js

```diff
@@ -391,25 +388,9 @@ function setupKeyListeners() {
     });
 };
 
-
-
-/*
-Process of determining which style to use. 
-TODO: Redo this using CSS variable or something. 
-Problem is that highlighter and shadow need to be in the same ID;
-not scalable right now to more customizable settings that effect the tracker. 
-*/
-
-// var trackerStyle = "trackerHighlighter"+highlighterSetting+"Shadow"+shadowSetting; // TODO: Refactor this color selection process
-
 // Classname for keyword highlights.
 
 function initializeTracker(settingsCustomizations) {
```

Comment:
is this method even necessary anymore?

Reply:
It's used to intialize the tracker in the beginning with the right settings, but functionality can probably be factored out elsewhere